tests/unit/lms/services/jstor_test.py

```diff
@@ -115,15 +120,46 @@ def test_metadata_formats_title(
         self, svc, http_service, api_response, expected_title
     ):
         http_service.get.return_value = factories.requests.Response(
-            json_data=api_response
+            json_data={**METADATA_RESPONSE_DEFAULTS, **api_response}
         )
+
         metadata = svc.metadata("12345")
+
         assert metadata["title"] == expected_title
 
-    def test_metadata_raises_if_schema_mismatch(self, svc, http_service):
-        invalid_api_response = {"title": ["This should be a string"]}
+    @pytest.mark.parametrize(
+        "has_pdf, access_level, expected_status",
+        [
+            (True, "full_access", ContentStatus.AVAILABLE),
+            (False, "full_access", ContentStatus.NO_CONTENT),
+            (True, "preview_access", ContentStatus.NO_ACCESS),
+        ],
+    )
+    def test_metadata_returns_content_status(
+        self, svc, http_service, has_pdf, access_level, expected_status
+    ):
+        api_response = {"has_pdf": has_pdf, "requestor_access_level": access_level}
         http_service.get.return_value = factories.requests.Response(
-            json_data=invalid_api_response
+            json_data=api_response
+        )
+        metadata = svc.metadata("12345")
+        assert metadata["content_status"] == expected_status
+
+    @pytest.mark.parametrize(
+        "api_response",
+        [
+            (
+                {
+                    "title": ["This should be a string"],
+                    **METADATA_RESPONSE_DEFAULTS,
+                }
+            ),
+            ({"title": "Test with missing fields"}),
+        ],
```

Comment:
```suggestion
        [
            {"title": ["This should be a string"], **METADATA_RESPONSE_DEFAULTS},
            {"title": "Test with missing fields"},
        ],
```

I think these are the same?